instagram-app/src/App.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './App.css';
+import PostsPage from './components/PostContainer/PostsPage';
+import withAuthenticate from './components/authentication/withAuthenticate';
+import Login from './components/Login/Login';
+
+const ComponentFromWithAuthenticate = withAuthenticate(PostsPage)(Login);
+
+
+class App extends React.Component {
+  /*constructor() {
```

Comment:
You can convert your App component to a functional component since you don't need state anymore.